lib/src/widgets/detailCardsWidgets/custom_appbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class CustomAppBar extends StatelessWidget {
+  const CustomAppBar({
+    Key? key,
+    this.movieImage,
+  }) : super(key: key);
+
+  final String? movieImage;
```

Comment:
why did you make this element nullable?, you have a default value for the image, it shouldn't be null at this point

Reply:
got it , symbol removed.